src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cells-gap;
+
+  margin: 0 auto;
+  padding: ($calendar-padding + $translate-length) $calendar-padding;
+  max-width: getMaxWidth(
+    $cell-size,
+    $cells-per-line,
+    $cells-gap,
+    $calendar-padding
+  );
+
+  $week:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $day, $pozition in $week {
+    &--start-day--#{$day} {
+      & :first-child {
+        margin-left: $pozition * ($cell-size + $cells-gap);
+      }
+    }
+  }
+
+  @for $month-length from 28 to 31 {
```

Comment:
Change the loop range to '@for $month-length from 28 through 31' to include 31 in the loop.